pywbemtools/pywbemlistener/_cmd_listener.py

```diff
@@ -449,6 +470,11 @@ def listener_list(context):
               required=False, default=1,
               help=u'Count of test indications to send. '
               'Default: 1')
+@click.option('-l', '--listener', type=str, metavar='LISTENER HOST',
```

Comment:
On `metavar='LISTENER HOST'`: The blank suggests these are two parameters. I suggest changing that to just "HOST".

Reply:
Done